src/client/game/states/splash.ts

```diff
@@ -35,7 +32,7 @@ export class SplashState extends Phaser.State {
       'zeldamap', 
       /* 'assets/tilemaps/zeldamap.json', */
       null,
-      processGameBoard(getGameState().gameBoard), 
+      boardToCSV(getGameState().gameBoard), 
```

Comment:
I wonder if we could have had the server / node send you the data in CSV format to begin with. Maybe not since Jon is also using it. Also saves the server some work having to convert it I suppose.